cypress/cypress.d.ts

```diff
@@ -13,7 +13,7 @@ declare global {
 
     interface Chainable {
       selection(subject: string, fn: any): Chainable;
-      setSelection(subject: string, query: string | object, endQuery: any[]): Chainable;
+      setSelection(subject: string, query?: string | object, endQuery?: any[]): Chainable;
```

Comment:
Given the query params are optional now, maybe can you remove a couple of //@ts-ignore in other uses of the function?